### PR TITLE
Small UI Fixes

### PR DESCRIPTION
**Pull request by ohamz**
### First draft of small fixes needed to be done:

- [x]  Modify BottomNavigationBar color to match color template.
- [x]  Adapt the Rating Icon in Landing Page (Recipe card).
- [x]  Adapt the Save Icon in Landing Page (Recipe card).
- [x]  Adapt Icons in Camera Preview.

Related issue : https://github.com/feedme-swent-epfl/feedme-android/issues/99#issue-2264064902

### More fixes:

- [x] Change the BottomBar Icons and remove labels to match the figma.
- [x] Fix UI of SmallThumbnailsDisplay (Icons and layout).
- [x] Adjust similar Icons among screens to stay consistent.
- [x] Change the second screen from Explore to Saved and adapt it in the code.
- [x] Change the middle screen form Create to Find Recipe and adapt the code.
- [x] SearchBar: Change Icons and Text.
- [x] Add goBack option when viewing a profile and change color of Follow button.

### After review:

- [x] Fix comments.
- [x] Add padding to selected page in bottom menu.

### CODE REVIEW

**Comment by ohamz** [comment]
More fixes:

- [ ] Change the BottomBar Icons and remove labels to match the figma
- [ ] Fix UI of SmallThumbnailsDisplay (Icons and layout)

**Comment by ohamz** [comment]
Will check the changes made later to fix the tests, probably linked to the Text composables.

**Comment by ohamz** [comment]
This PR is ready for review.

**Review by FacerOfGod — CHANGES_REQUESTED** [review]
The fixes appear to be well executed. They're certainly going to enhance the user experience. One minor aesthetic tweak I'd suggest is adding a bit more padding to the white box when a button is selected because it is currently touching the edge of the screen. If this requires a lot of effort, we can disregard it for now and do a fix later. Other than that great work! :D 

image

File: app/src/main/java/com/android/feedme/ui/SavedScreen.kt
Comment (mche3kek) [inline app/src/main/java/com/android/feedme/ui/SavedScreen.kt:1]: It is only the UI, the functionality is still not enabled right?

**Review by mche3kek — CHANGES_REQUESTED** [review]
Very nice improvements to the UI and the code, small comments and variables to check to be consistent across all files and I am unsure about the new name for the create screen.

Great job overall !

**Comment by ohamz** [comment]
> I'd suggest is adding a bit more padding to the white box when a button is selected because it is currently touching the edge of the screen. If this requires a lot of effort, we can disregard it for now and do a fix later. Other than that great work! :D

I'll work on it.

File: app/src/main/java/com/android/feedme/ui/SavedScreen.kt
Comment (ohamz) [reply app/src/main/java/com/android/feedme/ui/SavedScreen.kt:1]: Yes

File: app/src/androidTest/java/com/android/feedme/screen/FindRecipeScreen.kt
[the remark this answers is not in the document]
Comment (ohamz) [reply app/src/androidTest/java/com/android/feedme/screen/FindRecipeScreen.kt:1]: We can decide in a few hours during the Standup meeting.

**Review by mche3kek — APPROVED** [review]

**Comment by mche3kek** [comment]
Good job with correcting the errors, we will discuss the terminology of find_recipe with the team and fix it if needed in another PR

**Review by FacerOfGod — APPROVED** [review]

**Pull request merged**